packages/react-graphql/package.json

```diff
@@ -33,7 +33,7 @@
     "apollo-client": "^2.6.0",
     "apollo-link-http": "^1.5.15",
     "graphql-typed": "^0.4.0",
-    "react-apollo": ">=2.2.3 <3.0.0"
+    "react-apollo": ">=2.2.3 <4.0.0"
```

Comment:
Is the bottom end of this range still supported? (as in, not `>=3.0.0 <4.0.0`)

Reply:
The library itself is still supported, types are maybe a bit wonky but I can't select a more restrictive dependency for the types themselves :/